Game/Game/Game.cs

```diff
@@ -0,0 +1,187 @@
+﻿namespace Game;
+
+/// <summary>
+/// Class of simple game.
+/// </summary>
+public class Game
+{
+    /// <summary>
+    /// Width of the game map.
+    /// </summary>
+    private int width;
+    public int Width => width;
```

Comment:
Опять-таки, автосвойства.
```suggestion
    public int Width { get; };
```